scripts/sexbound/plugins/climax/climax.lua

```diff
@@ -557,13 +557,31 @@ function Sexbound.Actor.Climax:tryAutoClimax()
         return
     end
 
-    if self._config.currentPoints >= self:getThreshold() then
+    if self:getCurrentPoints() >= self:getThreshold() then
         self:beginClimax()
-
         self:refreshClimaxThreshold()
+
+        if math.random() <= self:getClimaxChainChance() then
+            self:getLog():debug("Actor " .. self:getParent():getActorNumber() .. " trying to trigger climax chain")
+
+            for _, actor in ipairs(self:getParent():getParent():getActors()) do
+                if actor:getActorNumber() ~= self:getParent():getActorNumber() then
+                    local plug = actor:getPlugins("climax")
+                    if plug and plug:getCurrentPoints() >= (plug:getThreshold() * 0.8) then
```

Comment:
Requires differentiation between players and NPCs/monsters, otherwise NPCs will just force players to climax aswell.

Reply:
I'm not referring to the one initiating the climax, but the other actors pulled along.
That the auto climax trigger only runs for NPCs I know, but the loop checking *all other actors* for that chain effect does not differentiate and triggers as long as that other actor is past the threshold.

This code should exclude players aswell.